docs/openapi/oad/ephemeral.oad.yaml

```diff
@@ -0,0 +1,80 @@
+openapi: 3.1.0
+info:
+  title: Ephemeral Service API
+  description: |
+    This API allows to interact with the Ephemeral service. It provides a single endpoint used to trigger computations.
+  version: 0.1.0
+  license:
+    name: Apache-2.0
+    url: https://github.com/carbynestack/ephemeral/blob/7ce9f830de3c5852afdc6c3c6c591fc5d04a50f7/LICENSE
+  contact:
+    name: Carbyne Stack
+    url: https://carbynestack.io/
+paths:
+  /:
+    post:
+      summary: Run a computation
+      description: |
+        Start a disctributed computation. The result of the computation is stored as a new Amphora Secret referenced
+        by the gameID provided when triggering the computation.
+      operationId: activate
+      tags:
+        - ephemeral
+      parameters:
+        - name: compile
+          description: |
+            Indicates whether *new* mpc program code is provided with this request. If a program is provided, ot will
```

Comment:
mpc -> MPC

Reply:
Done for all occurrences (for ephemeral, castor and amphora)